pkg/api/handlers/libpod/kube.go

```diff
@@ -89,21 +90,22 @@ func KubePlay(w http.ResponseWriter, r *http.Request) {
 	options := entities.PlayKubeOptions{
 		Annotations:        query.Annotations,
 		Authfile:           authfile,
-		Username:           username,
-		Password:           password,
+		IsRemote:           true,
+		LogDriver:          logDriver,
+		LogOptions:         query.LogOptions,
 		Networks:           query.Network,
 		NoHosts:            query.NoHosts,
+		Password:           password,
+		PublishPorts:       query.PublishPorts,
 		Quiet:              true,
-		LogDriver:          logDriver,
-		LogOptions:         query.LogOptions,
+		Replace:            query.Replace,
+		ServiceContainer:   query.ServiceContainer,
 		StaticIPs:          staticIPs,
 		StaticMACs:         staticMACs,
-		IsRemote:           true,
+		UseLongAnnotations: query.NoTrunc,
+		Username:           username,
 		Userns:             query.Userns,
```

Comment:
I like the alpha ordering, but man, GitHub doesn't make it easy to review.

Reply:
IMO such changes should be part of a separate commit.  That'll reduce the noise.